Cargo.toml

```diff
@@ -1,14 +1,55 @@
 [workspace]
-
+resolver = "2"
```

Comment:
This is the default if `edition = "2021"` isn't it?

Reply:
It is afaik, however without explicitly defining it at the workspace level I get

```
warning: some crates are on edition 2021 which defaults to `resolver = "2"`, but virtual workspaces default to `resolver = "1"`
note: to keep the current resolver, specify `workspace.resolver = "1"` in the workspace root's manifest
note: to use the edition 2021 resolver, specify `workspace.resolver = "2"` in the workspace root's manifest
```